bindings/wasm/src/account/update/attach_method_relationship.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use js_sys::Promise;
+use serde_repr::Deserialize_repr;
+use serde_repr::Serialize_repr;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::future_to_promise;
+
+use identity::account::Update;
+use identity::account::UpdateError::MissingRequiredField;
+use identity::core::OneOrMany::Many;
+use identity::core::OneOrMany::One;
+use identity::did::MethodRelationship;
+
+use crate::account::wasm_account::WasmAccount;
+use crate::error::wasm_error;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+#[wasm_bindgen(js_class = Account)]
+impl WasmAccount {
+  /// Attaches a single or multiple relationships to the given method, if the method exists.
+  ///
+  /// Note: The method needs to be in the set of verification methods,
+  /// so it cannot be an embedded one.
+  #[wasm_bindgen(js_name = attachMethodRelationships)]
+  pub fn attach_relationships(&mut self, input: &AttachMethodRelationshipOptions) -> Result<Promise> {
+    let relationships: Vec<WasmMethodRelationship> = match input.relationships().into_serde().wasm_result()? {
+      One(r) => vec![r],
+      Many(r) => r,
+    };
+
+    let relationships: Vec<MethodRelationship> = relationships.into_iter().map(Into::into).collect();
+
+    let account = self.0.clone();
+    let fragment = match input.fragment() {
+      Some(value) => value,
+      None => return Err(wasm_error(MissingRequiredField("fragment"))),
+    };
```

Comment:
I added a check for when the resulted `Vec` is empty, I'm not sure how to serialize to `Vec` without using `OneOrMany`. This is the reason we used it in the first place. But now, if types are used correctly, it's hard to mess things up from the TS side.

Reply:
No, you're right. The relationships need to be deserialized to `OneOrMany` first. My bad.

The empty check is still necessary in case someone passes an empty `Vec`, e.g. `[]`, which currently deserializes to `Many` I believe.